src/js/components/JSONEditor.js

```diff
@@ -407,7 +407,9 @@ class JSONEditor extends React.Component {
       ];
     }
 
-    return this.externalErrors.map(error => {
+    const errors = [].concat.apply([], this.externalErrors);
```

Comment:
Why this change? It's already doing `(this.props.errors || []).slice();` whenever it's set.

Reply:
@ahoskins please see my comment above https://github.com/dcos/dcos-ui/pull/2539/files#r161327729